src/components/HttpClient.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+
+export default class HttpClient {
+  sanitizeUrl = (url) => url.replaceAll('"', '').replaceAll(';', '');
+
+  getAsync = async (url) => {
+    /*
+    const API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.searchQuery}&format=json`;
+    const res = await axios.get(API);
+    */
+    return await axios.get(
+      this.sanitizeUrl(url)
+    );
+  };
+}

```

Comment:
I like where you're going with this as I created an HTTPClient class recently when I had to use fetch for a project instead of Axios.

If you want to separate your requests (which I think is a great idea), you can perhaps just create a separate file that includes functions to make the separate requests. 

Then, import those functions where you need to use them.